pcdsdevices/mv_interface.py

```diff
@@ -77,11 +84,86 @@ def camonitor(self):
         Updates current position of the motor.
         """
         try:
-            while True:
+            self._mov_ev.clear()
+            while not self._mov_ev.is_set():
                 print("\r {0:4f}".format(self.position), end=" ")
-                time.sleep(0.1)
+                self._mov_ev.wait(0.1)
         except KeyboardInterrupt:
             pass
+        finally:
+            self._mov_ev.clear()
+
+    def _stop_monitor(self):
+        self._mov_ev.set()
+
+    def Tweak_base(*args):
+        """
+        Base function to tweak motors
+        """
+        up = '\x1b[A'
+        down = '\x1b[B'
+        left = '\x1b[D'
+        right = '\x1b[C'
+        shift_up = '\x1b[1;2A'
+        shift_down = '\x1b[1;2B'
```

Comment:
don't have to change it now, but maybe in the future try to refer to `utils.up`, `utils.down` to avoid potential copy paste errors